src/hotspot/share/nmt/nmtPreInit.hpp

```diff
@@ -30,7 +30,7 @@
 #ifdef ASSERT
 #include "runtime/atomic.hpp"
 #endif
-#include "services/memTracker.hpp"
+#include "nmt/memTracker.hpp"
```

Comment:
The ASSERT Section should be moved down to after the main include block, or even better just include it without the guard. Might be nice to fix this while you are moving around the includes.